packages/table/src/types/ReactTable.ts

```diff
@@ -0,0 +1,69 @@
+import {
```

Comment:
The types are not correct from React-Table, so the go to recommendation from the Google is to create our own extensions of them. This is what this file is doing.